src/main/java/core/basesyntax/EmployeeService.java

```diff
@@ -1,10 +1,24 @@
 package core.basesyntax;
 
+import java.util.Comparator;
 import java.util.List;
 import java.util.Set;
+import java.util.TreeSet;
 
 public class EmployeeService {
     public Set<Employee> getEmployByOrder(List<Employee> employees) {
-        return null;
+        Set<Employee> employeeSet = new TreeSet<>(employeeComparator());
+        employeeSet.addAll(employees);
+        return employeeSet;
+    }
+
+    private Comparator<Employee> employeeComparator() {
+        return (o1, o2) -> {
```

Comment:
The 'employeeComparator' method should be static since it does not depend on any instance variables of 'EmployeeService'. Making it static would make it clear that its behavior is not tied to the state of 'EmployeeService' instances.